app/controllers/admin/poll/questions/answers_controller.rb

```diff
@@ -44,7 +50,4 @@ def load_answer
       @answer = ::Poll::Question::Answer.find(params[:id] || params[:answer_id])
     end
 
-    def load_answer
```

Comment:
Why is this being deleted but the `before_action` stays? We need this to show, edit and update the answer when requested.

Reply:
It's still there, it was duplicated
https://github.com/wairbut-m2c/consul/blob/23ef9ded1d02dcbf4304bd8eb89cd304d0f29156/app/controllers/admin/poll/questions/answers_controller.rb#L49

Reply:
Oh, sorry about that.

Reply:
no problem, appreciated it!